### DISCUSSION TITLE
App Manager install

### DISCUSSION THREAD

**Discussion by Vaflor** [HotCakeX/Harden-Windows-Security#434] [General]
Hi
I tried to install AppControl Manager but is says its root cert is expired.

![Rootcert](https://github.com/user-attachments/assets/03505080-e612-4e03-b12f-03e42da0418c)

Thanks

**Comment by HotCakeX** [comment]
On the readme page, release notes and the main page I've explained how to install it

https://github.com/HotCakeX/Harden-Windows-Security/wiki/AppControl-Manager#how-to-install-or-update-the-app

**Comment by Vaflor** [comment]
Yeah - I tried the powershell install method 
![image](https://github.com/user-attachments/assets/88cc6057-4668-451a-98eb-f353119ebeb9)

**Reply by HotCakeX** [reply]
Yes, as the message says you need to update your OS to the latest version because the app depends on new features that aren't available in older versions. You can check for updates in Windows update or use the update assistant here on Microsoft website: https://www.microsoft.com/en-us/software-download/windows11

the required version is 24H2 but you're on 23H2

**Reply by Vaflor** [reply]
Ahh, crap I cant update because of Safeguards Hold.
Also the version said 10.0.26100 so I thought I was good. 

Thanks for the quick support.

**Reply by HotCakeX** [reply]
ofc no problem, do you know the exact reason for the hold? 24H2 was released 2 months ago and had multiple cumulative updates since then

**Reply by Vaflor** [reply]
https://learn.microsoft.com/en-us/windows/release-health/status-windows-11-24h2#3424msgdesc

I'm on a Asus device .

Im Spinning up a VM with 24H2 so it wont stop me ;o)

**Reply by HotCakeX** [reply]
Thanks for the link, there are a couple of specific holds, the main issue with 23H2 and earlier is that policy removal or signed kernel-mode policy deployments require a reboot which isn't the case on 24H2 and later. I can try to make the app compatible with 23H2 so you can still use other features.

**Reply by Vaflor** [reply]
Thanks - you dont have to do that. I have a VM now .

Much appreciated thou.

**Reply by HotCakeX** [reply]
It's okay, i'm thinking about others with similar holds who want to try out the app and i don't want using a VM to become a requirement, so i made this issue based on this discussion https://github.com/HotCakeX/Harden-Windows-Security/issues/435

**Comment by HotCakeX** [comment]
@Vaflor You can now install AppControl Manager on Windows 11 23H2
https://github.com/HotCakeX/Harden-Windows-Security/releases/tag/AppControlManager.v.1.5.2.0

**Reply by Vaflor** [reply]
And its working. Seriously - you're awesome!

Thanks a bunch for this!
